pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -368,6 +414,24 @@ func (builder *podEntityDTOBuilder) getPodProperties(pod *api.Pod) ([]*proto.Ent
 	}
 	properties = append(properties, stitchingProperty)
 
+	if len(vols) > 0 {
+		var apiVols []*v1.PersistentVolume
+		for _, vol := range vols {
+			apiVols = append(apiVols, vol.UsedVolume)
+		}
+
+		m := stitching.NewVolumeStitchingManager()
+		err := m.ProcessVolumes(apiVols)
+		if err == nil {
```

Comment:
Would be helpful to log the err if err != nil

Reply:
Actually `ProcessVolumes` is logging the error if one is encountered internally. I am using the error returned from there only as a flag and retaining it as type error for any future use.

Reply:
This makes sense. Sorry for missing the errors in `ProcessVolumes` func.